### PR TITLE
configure read write user store doc and admin change in read only ldap

### PR DESCRIPTION
**Pull request by Buddhimah**

### CODE REVIEW

File: en/docs/learn/configuring-a-read-write-ldap-user-store.md [hunk 1]
```diff
@@ -1,21 +1,41 @@
...
-!!! tip "Before you begin!"
-    
-    Please read following topics:
-    
-    -   [Configuring User Stores](../../learn/configuring-user-stores)
-    -   [Configuring the Primary User
-        Store](../../learn/configuring-the-primary-user-store)
-    -   [Configuring the Secondary User
-        Store](../../learn/configuring-secondary-user-stores)
+WSO2 identity server uses a embedded Read/Write LDAP as the primary user store.
```
Comment (ayshsandu) [inline en/docs/learn/configuring-a-read-write-ldap-user-store.md]: an embedded

File: en/docs/learn/configuring-a-read-write-ldap-user-store.md [hunk 2]
```diff
@@ -1,21 +1,41 @@
...
+!!! tip 
+    Please read the topic [Configuring User Stores](../../learn/configuring-user-stores)  to get a high-level understanding of the user stores available in WSO2
+    Identity Server (WSO2 IS).
     
+## Configuring Read-write LDAP user store manager
 
-Read-write LDAP user store manager configured with
-**`          org.wso2.carbon.user.core.ldap.ReadWriteLDAPUserStoreManager         `**
-user store manager class.
+Following are the minimum configurations that are need to be provided to configure Read-only LDAP user store manager.
```
Comment (ayshsandu) [inline en/docs/learn/configuring-a-read-write-ldap-user-store.md]: are needed

File: en/docs/learn/configuring-a-read-write-ldap-user-store.md (hunk 1 above)
Comment (Buddhimah) [reply en/docs/learn/configuring-a-read-write-ldap-user-store.md]: done

File: en/docs/learn/configuring-a-read-write-ldap-user-store.md (hunk 2 above)
Comment (Buddhimah) [reply en/docs/learn/configuring-a-read-write-ldap-user-store.md]: done

**Pull request merged**